src/sim/TrackInitData.hh

```diff
@@ -57,22 +57,22 @@ struct TrackInitParamsData<W, MemSpace::host>
     //// DATA ////
 
     Items<Primary> primaries;     //!< Primary particles
-    size_type storage_factor = 3; //!< Initializer/parent storage per track
+    size_type      capacity = 3;  //!< Initializer/parent storage per track
```

Comment:
Maybe just remove the default value here.

Reply:
Hah, yes. Three is not a very good choice 🙄